src/serac/numerics/functional/boundary_integral_kernels.hpp

```diff
@@ -7,9 +7,12 @@
 
 #include <array>
 
+#include "serac/serac_config.hpp"
 #include "serac/numerics/functional/quadrature_data.hpp"
 #include "serac/numerics/functional/differentiate_wrt.hpp"
 
+#include "RAJA/RAJA.hpp"
```

Comment:
Since RAJA is a TPL, shouldn't it be
```c++
#include <RAJA/RAJA.hpp>
```
instead of double quotes?

Reply:
Our general style is to use `<>` for system libraries.